packages/atomic/src/components/atomic-result-placeholder-v1/atomic-result-placeholder.tsx

```diff
@@ -10,27 +10,17 @@ const placeholderClasses = 'block bg-neutral w-full h-full rounded';
 
 /**
  * The `atomic-result-placeholder` component provides an intermediate visual state that is rendered before the first results are available.
+ * @internal
  */
 @Component({
   tag: 'atomic-result-placeholder-v1',
   styleUrl: 'atomic-result-placeholder.pcss',
   shadow: true,
 })
 export class AtomicResultPlaceholder {
-  /**
-   * How results should be displayed.
-   */
-  @Prop() display: ResultDisplayLayout = 'list';
-
-  /**
-   * How large or small results should be.
-   */
-  @Prop() density: ResultDisplayDensity = 'normal';
-
-  /**
-   * How large or small the visual section of results should be.
-   */
-  @Prop() image: ResultDisplayImageSize = 'icon';
+  @Prop() display!: ResultDisplayLayout;
+  @Prop() density!: ResultDisplayDensity;
+  @Prop() image!: ResultDisplayImageSize;
```

Comment:
These comments weren't necessary so I removed them.

Reply:
Why do you say they weren't necessary ?

Reply:
Because the placeholder component is internal.